lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -3855,10 +3843,12 @@ Value FIRRTLLowering::createValueWithMuxAnnotation(Operation *op, bool isMux2) {
     builder.setInsertionPoint(op);
     StringRef namehint = isMux2 ? "mux2cell_in" : "mux4cell_in";
     for (auto [idx, operand] : llvm::enumerate(op->getOperands())) {
-      auto sym = moduleNamespace.newName(Twine("__") + theModule.getName() +
-                                         Twine("__MUX__PRAGMA"));
+      auto innerSym = hw::InnerSymAttr::get(StringAttr::get(
```

Comment:
Should(n't) this be using the `lowerInnerSymbol` method too?

Reply:
(Oh, is it not because the thing doesn't support inner symbols as it's an expression?)

Reply:
Yeah this is roughly because the input in this case does not already have an inner symbol.  I re-worked this code a bit to go through the new `getOrAddInnerSym` function to create a new inner symbol attribute as well.